server/audit_math/suite.py

```diff
@@ -1,718 +0,0 @@
-
-    # Add fake ballots to the hypothetical sample:
-    if prev_sample_size == 0:
-        # If no ballots have been sampled, assume the sample is roughly
-        # the margin
-        hyp_sample["hyp_round"] = {
-            winner: min(
-                int(n2 * (bp_stratum.vote_totals[winner] / bp_stratum.num_ballots)),
-                bp_stratum.vote_totals[winner],
-            ),
-            loser: min(
-                math.ceil(
-                    n2 * (bp_stratum.vote_totals[loser] / bp_stratum.num_ballots)
-                ),
-                bp_stratum.vote_totals[loser],
-            ),
-        }
-    else:
-        # Otherwise use the sample we've seen so far
-        hyp_sample["hyp_round"] = {
-            winner: min(
-                int(
-                    (n2 - prev_sample_size)
-                    * (cumulative_sample[winner] / prev_sample_size)
-                ),
-                bp_stratum.vote_totals[winner],
-            ),
-            loser: min(
-                math.ceil(
-                    (n2 - prev_sample_size)
-                    * (cumulative_sample[loser] / prev_sample_size)
-                ),
-                bp_stratum.vote_totals[loser],
-            ),
-        }
-
-    hyp_sample_size = n2
-    hyp_sample_size = min(bp_stratum.num_ballots, hyp_sample_size)
-
-    for rnd in bp_stratum.sample:
-        hyp_sample[rnd] = bp_stratum.sample[rnd]
-
-    hyp_no_cvr_stratum = BallotPollingStratum(
-        bp_stratum.num_ballots, bp_stratum.vote_totals, hyp_sample, hyp_sample_size
-    )
-
-    return maximize_fisher_combined_pvalue(
-        alpha, contest, hyp_no_cvr_stratum, hyp_cvr_stratum, winner, loser
-    )
-
-
-def get_sample_size(
-    risk_limit: int,
-    contest: Contest,
-    bp_stratum: BallotPollingStratum,
-    cvr_stratum: BallotComparisonStratum,
-) -> HybridPair:
-    """
-    Estimate the initial sample sizes for the audit.
-
-    Inputs:
-        risk_limit      - the risk limit (as an integer percentage) for this audit
-        contest         - the overall contest information
-        bp_stratum: a stratum object containing the ballot polling stratum
-        cvr_stratum: a stratum object containing the ballot comparison stratum
-    Outputs:
-        sample_sizes    - A Tuple of (cvr_strata_size, no_cvr_strata_size).
-    """
-
-    alpha = float(risk_limit) / 100
-
-    for winner, loser in product(contest.winners, contest.losers):
-        n_ratio = cvr_stratum.num_ballots / (
-            cvr_stratum.num_ballots + bp_stratum.num_ballots
-        )
-        num_sampled = max(
-            MIN_SAMPLE_SIZE, cvr_stratum.sample_size + bp_stratum.sample_size
-        )
-
-        ballots_to_sample = num_sampled
-
-        expected_pvalue = 1.0
-
-        # this allows us to exactly match CORLA18's estimate_n and estimate_escalation_n
-        coefficient = 1.1
-        if bp_stratum.sample_size == 0 and cvr_stratum.sample_size == 0:
-            coefficient = 2.0
```

Comment:
do you think the use of different coefficients was intentional? or just a typo or something?

Reply:
I think it's intentional. By limiting it for subsequent rounds you do your best to get lower sample sizes?